### PR TITLE
Fixed libssl being build by aws-lc, pruned bloat from wheel manifest

### PR DESCRIPTION
**Pull request by justinboswell**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -5,17 +5,17 @@ global-exclude .git*/**
...
 global-exclude crt/*/docs/**
 global-exclude docker-images/**
 prune crt/**/AWSCRTAndroidTestRunner
 prune crt/aws-c-auth/tests/aws-sig-v4-test-suite
 prune crt/aws-c-auth/tests/fuzz/corpus
-prune crt/s2n/tests/fuzz/corpus
-prune crt/s2n/tests/ctverif
-prune crt/s2n/tests/pems
-prune crt/s2n/tests/saw
-prune crt/s2n/tests/sidetrail
```
Comment (rccarper) [inline MANIFEST.in:18]: I guess s2n doesn't have these anymore because of aws-lc?

**Review by rccarper — APPROVED** [review]

File: MANIFEST.in (hunk 1 above)
Comment (justinboswell) [reply MANIFEST.in:18]: I just pruned the entire test suite, it's not necessary to reproduce the wheel or build from source

Comment (rccarper) [reply MANIFEST.in:18]: Oh, I see now, it just nukes the entire "tests" directory.  My mistake.

File: setup.py [hunk 2]
```diff
@@ -115,7 +115,7 @@ def __init__(self, name, extra_cmake_args=[], libname=None):
 # They're built along with the extension, in the order listed.
 AWS_LIBS = []
 if sys.platform != 'darwin' and sys.platform != 'win32':
-    AWS_LIBS.append(AwsLib('aws-lc', ['BUILD_LIBSSL=OFF'], 'crypto'))
+    AWS_LIBS.append(AwsLib('aws-lc', ['-DBUILD_LIBSSL=OFF'], 'crypto'))
```
Comment (graebm) [inline setup.py:118]: dddderp

**Pull request merged**